src/contracts/core/DelegationManagerStorage.sol

```diff
@@ -100,18 +112,32 @@ abstract contract DelegationManagerStorage is IDelegationManager {
      * @notice Minimum delay enforced by this contract per Strategy for completing queued withdrawals. Measured in blocks, and adjustable by this contract's owner,
      * up to a maximum of `MAX_WITHDRAWAL_DELAY_BLOCKS`. Minimum value is 0 (i.e. no delay enforced).
      */
-    mapping(IStrategy => uint256) public strategyWithdrawalDelayBlocks;
-
-    constructor(IStrategyManager _strategyManager, ISlasher _slasher, IEigenPodManager _eigenPodManager) {
+    mapping(IStrategy => uint256) private __deprecated_strategyWithdrawalDelayBlocks;
```

Comment:
Are we not going to have `strategyWithdrawalDelay` for custom strategy withdrawal delay timestamps?

Reply:
EIGEN is going to remain at a 7 day withdrawal delay nullifying the need for this mapping. For clarity @gpsanant